src/middlewares/auth-middleware.ts

```diff
@@ -48,104 +47,24 @@ async function authentication(
     }
     span.setDisclosedAttribute("user cookie found", true);
 
-    const { access_token: currentAccessToken, sub } = userCookie;
+    const { access_token: currentAccessToken } = userCookie;
 
     await decryptVerifyAccessToken(currentAccessToken).catch(async (error) => {
       if (error.name === "TokenExpiredError") {
         span.setDisclosedAttribute("is access_token expired", true);
 
-        const user = await getDBUserFromSub(sub).catch((error) => {
-          span.setDisclosedAttribute("database reachable", false);
-          throw webErrorFactory({
-            ...webErrors.databaseUnreachable,
-            parentError: error,
-          });
-        });
-        span.setDisclosedAttribute("database reachable", true);
-
-        if (!user) {
-          span.setDisclosedAttribute("user found on DB", false);
-          throw new NoDBUserFoundError();
+        if (request.url && request.url.includes("/api/")) {
+          throw webErrorFactory(webErrors.unauthorized);
         }
```

Comment:
Here, I filter the request to api pages, to avoid having multiple redirection at the end of refresh token flow. I throw an unauthorized web error, which will be catch in `_app`. (cf further comment)